### PR TITLE
Ginkgo/fixup

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

File: stanford/playbooks/ami.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ansible-playbook
+---
+- name: Create AMI
```
[the remark this answers is not in the document]
Comment (caesar2164) [reply stanford/playbooks/ami.yml]: nice!

File: stanford/playbooks/analytics-api.yml [hunk 2]
```diff
@@ -54,3 +57,5 @@
       apt: name=libffi-dev state=present
     - name: 'Install make'
       apt: name=make state=present
+- include: ../playbooks/ami.yml
+  when: redeploy | default(False) | bool
```
[the remark this answers is not in the document]
Comment (caesar2164) [reply stanford/playbooks/analytics-api.yml]: w00t!

File: stanford/suclass/group_vars/all [hunk 3]
```diff
@@ -10,6 +10,7 @@ EDXAPP_CMS_BASE: "studio.{{ EDXAPP_LMS_BASE }}"
 EDXAPP_CMS_ENABLE_BASIC_AUTH: True
 EDXAPP_CMS_NGINX_PORT: 80
 EDXAPP_FEATURES_SHOW_ABOUT_LINK: false
+EDXAPP_FOOTER_DISCLAIMER_TEXT: 'Stanford University, &copy; 2018'
```
Comment (stvstnfrd) [inline stanford/suclass/group_vars/all]: @caesar2164 This needs to be a literal copyright symbol, right?

**Comment by stvstnfrd** [comment]
@kluo @dcadams @caesar2164 I pushed a few more changes up here for production and cleaned up all of the commits.

I'm ready to merge this to master/rc.

**Pull request merged**